<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriveControlWidget</class>
 <widget class="QWidget" name="DriveControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DriveControlWidget</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Forward Velocity</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="spinBoxForwardVelocity">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-0.150000000000000</double>
     </property>
     <property name="maximum">
      <double>0.150000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Turning Rate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="spinBoxTurningRate">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-5.000000000000000</double>
     </property>
     <property name="maximum">
      <double>5.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Left Wheel Speed</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLCDNumber" name="valueLeftWheelSpeed">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Right Wheel Speed</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLCDNumber" name="valueRightWheelSpeed">
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Left Encoder</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLCDNumber" name="valueLeftEncoder">
     <property name="numDigits">
      <number>6</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Right Encoder</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLCDNumber" name="valueRightEncoder">
     <property name="numDigits">
      <number>6</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="pushButtonStart">
     <property name="text">
      <string>Set Velocity</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="pushButtonStop">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="pushButtonReset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
